cloud-formation/src/templates/cfn-template.yaml

```diff
@@ -120,4 +120,46 @@ Resources:
             {{> stateMachine}}
 
           - {}
-      RoleArn: !GetAtt [ StatesExecutionRole, Arn ]
\ No newline at end of file
+      RoleArn: !GetAtt [ StatesExecutionRole, Arn ]
+
+  NoStateMachineExecutionsAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: CreateProdResources
+    Properties:
+      InsufficientDataActions:
+      - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:simple-dev
+      OKActions:
```

Comment:
If this resolves itself (i.e. people start signing up again after a quiet period), we get emailed about that too